### PR TITLE
fix histogram for empty inputs & no data,

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Histogram-Operator on no data raster with range type data: Operator: DataTypeError: Plot exception: Histograms max value must be larger than its min value](https://trello.com/c/IB4pFpdL/155-histogram-operator-on-no-data-raster-with-range-type-data-operator-datatypeerror-plot-exception-histograms-max-value-must-be-lar)

File: operators/src/plot/histogram.rs [hunk 1]
```diff
@@ -313,6 +323,19 @@ impl HistogramRasterQueryProcessor {
 
         Ok(chart)
     }
+
+    fn empty_histogram(
+        &self,
+    ) -> Result<<HistogramRasterQueryProcessor as PlotQueryProcessor>::OutputFormat> {
+        let histogram =
+            geoengine_datatypes::plots::Histogram::builder(1, 0., 0., self.measurement.clone())
```
Comment (jdroenner) [inline operators/src/plot/histogram.rs:331]: Thats not working with the ui: 
"Operator: DataTypeError: Plot exception: Histograms max value must be larger than its min value"

File: operators/src/plot/histogram.rs [hunk 2]
```diff
@@ -384,6 +407,19 @@ impl HistogramVectorQueryProcessor {
...
```
Comment (jdroenner) [inline operators/src/plot/histogram.rs:415]: same here

File: operators/src/plot/histogram.rs [hunk 3]
```diff
@@ -393,6 +429,12 @@ struct HistogramMetadata {
     pub max: f64,
 }
 
+impl HistogramMetadata {
+    fn has_invalid_parameters(&self) -> bool {
+        self.number_of_buckets == 0 || self.min > self.max
```
Comment (jdroenner) [inline operators/src/plot/histogram.rs:434]: you need to check for min >= max i think

**Review by jdroenner — CHANGES_REQUESTED** [review]
sadly this does not work for the ui

File: operators/src/plot/histogram.rs (hunk 3 above)
Comment (ChristianBeilschmidt) [reply operators/src/plot/histogram.rs:445]: 359ff6d

File: operators/src/plot/histogram.rs (hunk 2 above)
Comment (ChristianBeilschmidt) [reply operators/src/plot/histogram.rs:418]: 359ff6d

File: operators/src/plot/histogram.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply operators/src/plot/histogram.rs:334]: 359ff6d

**Pull request merged**